### DISCUSSION TITLE
How to register the package file generated by graphics32 if....

### DISCUSSION THREAD

**Discussion by MaheshNaikM** [graphics32/graphics32#267] [General] [answered]
I want to use the ..280.bpl  package file generated by the delphi compiler in a CE c++ compiler.How do i register them in a new VM which has only CE edition of cbuilder v11.I require the timage32 design component.!   Resolved with help from this group.I just copy the files generated by the package into 2 groups ..one which contains all the .hpp files and the other which contains the 10 package(bpl,dcp,lib.) files --5 for D and 5 for R. i install the d file which contains the draw component before using in my cpp program....

**Comment by andersmelander** [comment, marked as answer]
I don't know how you do in CBuilder but in Delphi you can do it through the Component menu:
![billede](https://github.com/graphics32/graphics32/assets/18141921/c01782c0-f41f-4630-af6e-9314bfd85c46)
